lang/en.json

```diff
@@ -967,6 +967,12 @@
         "SiteWithoutDataMtmStep3": "Copy the code below and paste it on every page of your website. We recommend pasting it immediately before the closing %1$s tag. %2$sLearn more%3$s.",
         "IgnoreGtmDataLaterDescription": "If checked, Matomo Tag Manager excludes data layer values that were already set by Google Tag Manager. For compatibility reasons, the default behavior is to keep data layer values set by Google Tag Manager.",
         "IgnoreGtmDataLaterTitle": "Ignore Google Tag Manager data layer",
-        "VersionEditWithNoAccessMessage": "Adding or editing versions requires the \"Tag Manager: Use Custom Templates\" capability. If you believe you should have this capability, please contact your Matomo administrator. %1$sLearn more%2$s"
+        "VersionEditWithNoAccessMessage": "Adding or editing versions requires the \"Tag Manager: Use Custom Templates\" capability. If you believe you should have this capability, please contact your Matomo administrator. %1$sLearn more%2$s",
+        "MtmTrackingCodeIntro": "Installing Matomo Tag Manager is the best way to track data, as it allows you to modify tracking configurations  at any time without needing to alter your website.",
```

Comment:
The source of this text had a double space after the word "configurations" and it got copied in, although a browser would collapse it anyways.

Reply:
Thank you. I'll remove it.